scanner/src/main/java/ca/on/oicr/gsi/runscanner/scanner/processor/DefaultIllumina.java

```diff
@@ -443,11 +457,12 @@ public NotificationDto process(File runDirectory, TimeZone tz) throws IOExceptio
           Path baseCallDirectory =
               Paths.get(dto.getSequencerFolderPath(), "Data", "Intensities", "BaseCalls");
           // Check that each lane directory is complete
+          IlluminaNotificationDto finalDto = dto;
           boolean dataCopied =
               IntStream.rangeClosed(1, dto.getLaneCount()) //
                   .mapToObj(lane -> String.format("L%03d", lane)) //
                   .map(baseCallDirectory::resolve) //
-                  .allMatch(laneDir -> isLaneComplete(laneDir, dto));
+                  .allMatch(laneDir -> isLaneComplete(laneDir, finalDto));
```

Comment:
?

Reply:
Without this, build fails with
"[ERROR] Failed to execute goal org.apache.maven.plugins:maven-compiler-plugin:3.13.0:compile (default-compile) on project scanner: Compilation failure
[ERROR] /home/avarsava/Workspace/runscanner/scanner/src/main/java/ca/on/oicr/gsi/runscanner/scanner/processor/DefaultIllumina.java:[464,64] local variables referenced from a lambda expression must be final or effectively final"